router/core/graph_server.go

```diff
@@ -950,7 +950,7 @@ func (s *graphServer) buildGraphMux(ctx context.Context,
 					append([]attribute.KeyValue{
 						otel.WgOperationName.String(item.OperationName),
 						otel.WgClientName.String(item.ClientName),
-						otel.WgClientVersion.String(item.ClientVersion),
```

Comment:
Tbh this is a good reason to keep the version 😄 Because if the router always use the cache, we might never plan this query with client information anymore.

Reply:
Yeah, that's the thing I wanted to ask you yesterday.

Reply:
Done